lib/puppet/util/windows/api_types.rb

```diff
@@ -0,0 +1,278 @@
+require 'ffi'
+require 'puppet/util/windows/string'
+
+module Puppet::Util::Windows::APITypes
```

Comment:
Is there a plan to import the Puppet Gem into the PDK at some point?  If so, this looks like a namespacing issue.